src/hotspot/os_cpu/bsd_x86/os_bsd_x86.cpp

```diff
@@ -404,9 +404,12 @@ frame os::get_sender_for_C_frame(frame* fr) {
 }
 
 intptr_t* _get_previous_fp() {
-#if defined(SPARC_WORKS) || defined(__clang__) || defined(__llvm__)
+#ifdef SPARC_WORKS
```

Comment:
I don't see how SPARK_WORKS could ever be true here, so I'd suggest just removing this portion altogether